### PR TITLE
Create Onboarding.md

### PR DESCRIPTION
**Pull request by dmontauk**
Add instructions on how to add people to the Junto.

### CODE REVIEW

File: Onboarding.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Onboarding process
+
+After someone has been accepted into the Junto we need to sign the up for:
```
Comment (dmp42) [inline Onboarding.md]: "need to sign the up" > "need to sign up" (or "need to sign him/her up")

File: Onboarding.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+1. Slack: http://eng-junto.slack.com
+2. Github: http://github.com/eng-junto
```
Comment (dmp42) [inline Onboarding.md]: We don't necessarily _need_ to add them on github (though we can).

**Comment by dmp42** [comment]
Some nits - LGTM otherwise!

**Comment by dmontauk** [comment]
Updated.

**Pull request merged**